front/src/components/icons.tsx

```diff
@@ -91,3 +91,16 @@ export const MinusIcon = Iconsax.Minus;
 export const DoneIcon = Iconsax.TickSquare;
 
 export const EmptyStateIcon = Iconsax.SearchStatus1;
+
+// Icons for song type
+export const SongTypeOriginalIcon = Iconsax.Music;
+export const SongTypeRemixIcon = Iconsax.Repeat;
+export const SongTypeLiveIcon = Iconsax.Microphone2;
+export const SongTypeAcousticIcon = Iconsax.VoiceSquare;
+export const SongTypeInstrumentalIcon = Iconsax.Musicnote;
+export const SongTypeEditIcon = Iconsax.Cut;
+export const SongTypeCleanIcon = Iconsax.TickCircle;
+export const SongTypeDemoIcon = Iconsax.Edit;
```

Comment:
Using this icon feels weird, it looks like an edit icon (as if we could change the song type).
Could you use `ReceiptEdit` instead?